src/util/util.go

```diff
@@ -59,11 +61,11 @@ func PrintError(messages ...string) {
 	errHeaderFmt := color.New(color.Bold).Add(color.FgRed)
 	errMessageFmt := color.New(color.FgRed)
 	fmt.Println()
-	_, err := errHeaderFmt.Println("  Error")
-	exit.If(err)
+	// NOTE: no point checking the error here,
+	//       we are printing an error already.
```

Comment:
Disagree with this since we don't use `os.exit()` in this function

Reply:
Okay, added `os.Exit` back.

Reply:
Correction. Your feedback has made me think about this again. Now this falls back to printing without color in case printing with color is not possible. All we want to do is printing here. The colors not working is not a critical condition that would justify terminating the entire process.